### DISCUSSION TITLE
blog/cookbook-20230701-1

### DISCUSSION THREAD

**Discussion by giscus** [wjwei-handsome/wjwei-docusaurus#9] [General]
# blog/cookbook-20230701-1

学习一下家常菜，2023年7月1日午餐菜谱

https://www.wjwei.blog/blog/cookbook-20230701-1

**Comment by lovefc** [comment]
会做菜的男人，很有魅力！

**Reply by wjwei-handsome** [reply]
主打一个性价比！
